### PR TITLE
chore: update translation strings via Crowdin

### PR DESCRIPTION
**Pull request by gongracr**

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.0(5861231)(5861231)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.0(5861231)(5861231)-staging-compat.apk)

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -501,8 +501,8 @@
     <string name="label_system_message_team_member_left">%1$s wurde aus dem Team entfernt</string>
     <string name="label_system_message_user_not_available">Dieser Benutzer ist nicht mehr verfügbar</string>
     <string name="label_system_message_new_conversation_receipt_mode">Lesebestätigungen sind %1$s</string>
-    <string name="label_system_message_read_receipt_changed_by_self">%1$s haben Lesebestätigungen für alle %2$sgeschaltet</string>
-    <string name="label_system_message_read_receipt_changed_by_other">%1$s hat Lesebestätigungen für alle %2$sgeschaltet</string>
+    <string name="label_system_message_read_receipt_changed_by_self">%1$s turned read receipts %2$s for everyone</string>
+    <string name="label_system_message_read_receipt_changed_by_other">%1$s turned read receipts %2$s for everyone</string>
```
Comment (ohassine) [inline app/src/main/res/values-de/strings.xml]: English for DE is that correct ?

Comment (gongracr) [reply app/src/main/res/values-de/strings.xml]: wtf, I don't understand why this got mixed up with Crowdin 🤦 so no, it's broken. I will update it on their site and trigger a new one

Comment (gongracr) [reply app/src/main/res/values-de/strings.xml]: fixed

**Review by tmspzz — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.0(5861566)(5861566)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.0(5861566)(5861566)-staging-compat.apk)

**Review by ohassine — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5861917)(5861917)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5861917)(5861917)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/7/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5862145)(5862145)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5862145)(5862145)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [9](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/9/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5862535)(5862535)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5862535)(5862535)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [10](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/10/) **failed**.

**Comment by AndroidBob** [comment]
Build [12](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/12/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5863669)(5863669)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5863669)(5863669)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [13](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/13/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5870936)(5870936)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5870936)(5870936)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [14](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/14/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5872893)(5872893)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5872893)(5872893)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [16](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/16/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5931980)(5931980)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5931980)(5931980)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [17](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/17/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5932765)(5932765)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5932765)(5932765)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [18](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/18/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5948420)(5948420)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5948420)(5948420)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [19](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/19/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5948846)(5948846)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5948846)(5948846)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [20](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/20/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5973668)(5973668)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5973668)(5973668)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [22](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/22/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5974604)(5974604)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5974604)(5974604)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [23](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/23/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(5974927)(5974927)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(5974927)(5974927)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [25](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/25/) **failed**.

**Comment by AndroidBob** [comment]
Build [27](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/27/) **failed**.

**Comment by AndroidBob** [comment]
Build [28](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/28/) **failed**.

**Comment by AndroidBob** [comment]
Build [30](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/30/) **failed**.

**Comment by AndroidBob** [comment]
Build [31](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/31/) **failed**.

**Comment by AndroidBob** [comment]
Build [33](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/33/) **failed**.

**Comment by AndroidBob** [comment]
Build [35](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/35/) **failed**.

**Comment by AndroidBob** [comment]
Build [37](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/37/) **failed**.

**Comment by AndroidBob** [comment]
Build [38](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/38/) **failed**.

**Comment by AndroidBob** [comment]
Build [39](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/39/) **failed**.

**Comment by AndroidBob** [comment]
Build [41](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/41/) **failed**.

**Comment by AndroidBob** [comment]
Build [42](https://10.10.124.134/job/wire-android-reloaded/job/PR-1509/42/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(6052028)(6052028)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(6052028)(6052028)-staging-compat.apk)

**Pull request merged**